tools/clang/tools/dxildll/CMakeLists.txt

```diff
@@ -58,7 +58,7 @@ target_link_libraries(dxildll PRIVATE
   dxcvalidator
 )
 
-if (WIN32)
+if (WIN32 AND HLSL_BUILD_DXILCONV)
```

Comment:
DxcEtw is not used in dxildll.
We could just remove the if and always
add_dependencies(dxildll
    ${DXCDependencies}